server/controllers/items/last_public.coffee

```diff
@@ -7,7 +7,7 @@ bundleOwnersToItems = require './lib/bundle_owners_to_items'
 module.exports = (req, res)->
   { query } = req
   { limit, offset } = query
-  assertImage = query['assert-image'] is 'true'
+  assertImage = _.parseBooleanString query['assert-image']
```

Comment:
why lodash more than vanilla ? to me lodash is for complex manipulation, this shoudl stay as it was

Reply:
there's a confusion there: `parseBooleanString` isn't defined by lodash but as a [shared utils](https://github.com/inventaire/inventaire-client/blob/09d553d/app/lib/shared/utils.coffee#L74).
[Many utils functions are added on `_`](https://github.com/inventaire/inventaire/blob/b93c9c4/server/builders/utils.coffee#L16) without implementing Lodash chaining features or anything, just as a way to access those generalist functions easily, but it might be too much and could desired to be split to ease comprehension of where a given function is defined.

But back in the present case, I'm for using  `parseBooleanString` as, while not doing much, it gives the statement a "this is a common pattern" tag, which can then be searched through the code base more easily

Reply:
because that's the first time you encounter `parseBooleanString`, but [next time you will see it](https://github.com/inventaire/inventaire/search?utf8=%E2%9C%93&q=parseBooleanString&type=), the intent will be obvious

Reply:
that doesnt answer my remark, the intent is less obvious 
it also creates a different way to assign it, between `recent_public` and `last_public`, at least change it also on the  `recent_public`.
If the goal is to DRY those two in the future, and we write syntax differences just because "its already somewhere else in the code, so its fine", then this goal is getting a step further.

Reply:
that's already the case: [`recent_public` uses it too](https://github.com/inventaire/inventaire/blob/0d14635/server/controllers/items/recent_public.coffee#L27)

Reply:
to be honest, following lodash syntax rather than coffee, converges more toward es6 standards